new-client/src/plugins/Coordinates/CoordinatesView.js

```diff
@@ -81,53 +91,67 @@ class CoordinatesView extends React.PureComponent {
 
   render() {
     return (
-      <>
-        <StyledPaper>
-          <Table>
-            <TableHead>
-              <TableRow>
-                <TableCell>Projektion</TableCell>
-                <TableCell>Koordinater</TableCell>
-              </TableRow>
-            </TableHead>
-            <TableBody>{this.renderProjections()}</TableBody>
-          </Table>
-          <Grid container justifyContent="space-between">
-            <StyledButton
-              variant="text"
-              onClick={() => {
-                this.props.model.zoomOnMarker();
-              }}
-            >
-              Zooma
-            </StyledButton>
-            <StyledButton
-              variant="text"
-              onClick={() => {
-                this.props.model.centerOnMarker();
-              }}
-            >
-              Panorera
-            </StyledButton>
-            <StyledButton
-              variant="text"
-              onClick={() => {
-                this.props.model.goToUserLocation();
-              }}
-            >
-              Min position
-            </StyledButton>
-            <StyledButton
-              variant="text"
-              onClick={() => {
-                this.props.model.resetCoords();
-              }}
-            >
-              Rensa fält
-            </StyledButton>
+      <Root>
+        <StyledGridContainer container spacing={1}>
+          <Grid item>
+            <StyledGrid>Projektion</StyledGrid>
           </Grid>
-        </StyledPaper>
-      </>
+          <Grid item xs>
+            <StyledGrid>Koordinater</StyledGrid>
+          </Grid>
+        </StyledGridContainer>
+        {this.renderProjections()}
+        <Grid container spacing={1}>
+          <Grid item>
+            <Tooltip title="Rensa fält">
+              <StyledButton
+                onClick={() => {
+                  this.props.model.resetCoords();
+                }}
+              >
+                Rensa
+              </StyledButton>
+            </Tooltip>
+            <Tooltip title="Min position">
+              <StyledButton
+                onClick={() => {
+                  this.props.model.goToUserLocation();
+                }}
+              >
+                Min position
+              </StyledButton>
+            </Tooltip>
+          </Grid>
+          <Grid item xs>
+            <Grid container direction="row-reverse">
+              <Grid item>
```

Comment:
The selected rows are just an example - this comment is generic about the entire Grid you have in this View.

It looks to me as you try to have a [nested grid](https://mui.com/material-ui/react-grid/#nested-grid) but failed to created it correctly. So you end up with something like:
```
| item | item                 |
| another Item | another Item |
```
where you actually wanted:
```
| item         |  item        |
| another Item | another Item |
```

Take a look at MUI's documentation and how to create nested grids.